### ISSUE TITLE
pip installation is not working

### ISSUE THREAD

**Issue by mariamorabi** [acsicuib/YAFS#66]
I'm trying to install yafs on windows 10.
I have python 3.10.2, pip 22.2.2
I have all the requirements installed and ready but I'm struggling to build the yafs library

"pip install yafs" gives this error:
ERROR: Could not find a version that satisfies the requirement yafs (from versions: none)
ERROR: No matching distribution found for yafs

and the yaml file mentioned in the installation guide is missing in the files of version 3, so conda installation is not easy.

**Comment by wisaaco** [comment]
Hello,
You only need to clone the project (the specific branch) and include the YAFS library in your project using pythonpath environment. 
Take a look:
https://github.com/acsicuib/YAFS#installation
https://github.com/acsicuib/YAFS#getting-started
Best,
Isaac

**Comment by mariamorabi** [comment]
Hi! Thanks for the fast reply.

This is easy to do in Linux, but it's not that easy in Windows.
I had to follow this answer to add the directory https://stackoverflow.com/a/59411635 
It would've saved a lot of time if the pip command worked.

Thank you for your efforts.

**Comment by insooth** [comment]
Try this. Create a file `requirements.txt` with these contents:

```
-e git+https://github.com/acsicuib/YAFS@YAFS3#egg=yafs ; python_version >= "3.9.7"
```
Then run `pip install -r requirements.txt` which will download YAFS. Note that it raise an error during execution of `setup.py` (fixed with #68), however that can be safely ignored.
